widgets/image-grid/image-grid.php

```diff
@@ -134,7 +134,7 @@ function get_template_variables( $instance, $args ) {
 
 			if ( empty( $image['image'] ) && ! empty( $image['image_fallback'] ) ) {
 				$alt = ! empty ( $image['alt'] ) ? $image['alt'] .'"' : '';
-				$image['image_html'] = '<img src="'. esc_url( $image['image_fallback'] ) .'" alt="'. esc_attr( $alt ) .'" title="'. esc_attr( $title ) .'">';
+				$image['image_html'] = '<img src="'. esc_url( $image['image_fallback'] ) .'" alt="'. esc_attr( $alt ) .'" title="'. esc_attr( $title ) .'" class="sow-image-grid-image_html">';
```

Comment:
Due to the conditional this change is contained in, the class will only be added if a fallback is used and an image is not added via the media button.

Reply:
Oh sorry about that. I was using a prebuilt Layout and had forgotten to check this.